src/main/java/no/uib/inf112/core/map/tile/Attribute.java

```diff
@@ -52,6 +52,21 @@
      * Mark this movable tile to be pushed if something moves onto it
      */
     PUSHABLE(MovableTile.class),
+
+    /**
+     * This tile shoots a laser
+     */
+    SHOOTS_LASER,
+
+    /**
+     * This tile lays down laser, meaning it puts down laser on the entitylaser layer
+     * e.g robots
+     */
+    LAYS_DOWN_LASER,
+    /**
+     * This tile shoots two lasers
+     */
+    DOUBLE_LASER,
```

Comment:
Attributes should be as generic as possible. This can be replaced with HIGH_PRIORITY, as its exact meaning is up to the phase implementation

Reply:
But what when we in the future implements that some robots have double lasers, i'm afraid that it would become an issue if somewhere else uses High priority

Reply:
Currently HIGH_PRIORITY is not used by players, and I can't really see a scenario where we would do that.

Reply:
I would like the attributes to be as generic as possible, as said in the initial comment, and this is a great use for the HIGH_PRIORITY